walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/tennis.html

```diff
@@ -2,12 +2,27 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title> My Tennis Journey</title>
+    <title>My Tennis Journey</title>
     <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Tennis Journey</h1>
-    </div> 
+    </head>
+    <body>
+        <div id="content">
+            <h1>My Tennis Journey</h1>
+            <p>  
+                 I started playing tennis, when I was 8 years old, but stopped at 9.
+                 I didn't pick it back up until, I transferred 
+                 to the high school I graduated from in my sophmore year. I joined their tennis team in my sophmore
```

Comment:
nit: line 14 was broken too early. If you don't explicitly need a line return, I'd recommend to keep text on the line until before you reach 125 chars, so it is slightly more readable.

These are really small nitpicks, but they're standard practice, especially in professional project, so the codebase has a consistent styling and is easily readable by everyone.